rstudio/rstudio
# Make native pipe a project-specific option

### Intent

Address #9409

### Approach

Use of the native pipe operator should be an option on the Project Options level as well as the Global level, making it easier to disable the native pipe in projects that use older versions of R

To help user confusion around a project option that overrides a global option, we've added some text and a button to the global options editing pane, to remind the user that project options may conflict, and to take them over to project options when they hit the button.

### Automated Tests

None

### QA Notes

Set Global Option for "Use native pipe operator" to True via Options pane, set project option to False via Project options pane. Confirm user preferences are respected by entering 
Cmd + Shift + M in the console or editor (will insert magrittr or R native pipe depending on user preference)
Also run `.rs.readUiPref("insert_native_pipe_operator")` and `.rs.readUserPref("insert_native_pipe_operator")` to confirm

Be sure to play around with changing the Global option while project option is set to default, and changing the project option to override the project option. Option should only be written to Rproj file when project option is set to Yes or No, but not when set to default

Make sure changing the project option doesn't change the global option.

### Checklist
